API/NoDocs/Microsoft.Extensions.ObjectPool/ObjectPoolServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,16 @@
+// Assembly 'Microsoft.Extensions.ObjectPool.DependencyInjection'
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.ObjectPool;
```

Comment:
Should this be in `Microsoft.Extensions.DependencyInjection`?

Reply:
Done.